include/rocksdb/metadata.h

```diff
@@ -34,6 +35,8 @@ struct ColumnFamilyMetaData {
   std::string name;
   // The metadata of all levels in this column family.
   std::vector<LevelMetaData> levels;
+
+  std::vector<BlobMetaData> blobs;
```

Comment:
Let's call this `blob_files` to make its meaning clearer. Also please add the number and total size of blob files as fields.